app/Resources/views/blog/post_show.html.twig

```diff
@@ -32,7 +32,7 @@
         <div class="row post-comment">
             <h4 class="col-sm-3">
                 <strong>{{ comment.authorEmail }}</strong> {{ 'post.commented_on'|trans }}
-                <strong>{{ comment.publishedAt|date('M jS \\a\\t g:i a') }}</strong>
+                <strong>{{ comment.publishedAt|localizeddate('medium', 'short', app.request.get('_locale')|default('en')) }}</strong>
```

Comment:
We can't change it :(  We're using the Twig extension filter and it's called like that:

![twig_filter](https://cloud.githubusercontent.com/assets/73419/8783842/b70e1410-2f1e-11e5-8fad-fe532fa11240.png)